### ISSUE TITLE
Compatibility/ Inclusion of Advance Gaze into HoneySelectVR

### ISSUE THREAD

**Issue by Jake762x51** [Eusth/HoneySelectVR#30]
Can you please either include Advance Gaze(http://www.hongfire.com/forum/forum/hentai-lair/hf-modding-translation/honey-select-mods/5737526-advanced-gaze-for-vr) into HonySelectVR or make it at least compatible with the current build ?

**Comment by Trichinosis** [comment]
Just an additional comment here, I think it was slightly more immersive when the model would stare into the left eye.  Currently she's fixated on the bridge of your nose, which is slightly uncanny-valley / stepford wife-y.  When looking closely at someone, your eyes flick from eye to eye, left to right, and it would be great to add this oscillation somehow.  Advanced Gaze for VR doesn't currently do this, and the eyeballs float around a bit much for me.

**Comment by Jake762x51** [comment]
What you have to realise in reality you don't just look someone straight into the left or right eye and that's it you change between looking into the left, right eyes, then nose, mouth and back to the eyes again in very short intervals and sometimes this is something that Advanced Gaze tries to archieve
![where-to-look-social-gaze](https://cloud.githubusercontent.com/assets/24995644/21755723/72c49872-d618-11e6-9048-05fe30f47d4a.jpg) It is called triangular eye gaze.

**Comment by Trichinosis** [comment]
Yep, I get that.  But Advanced Gaze doesn't go between eyes, it just goes between the bridge of your nose and the tip of your nose, and the offset is a bit broken.  It tries but much work is needed to make it an actual triangular gaze.
